src/renderers/shaders/ShaderChunk/packing.glsl

```diff
@@ -11,7 +11,7 @@ vec4 packLinearUnitToRGBA( const in float value ) {
 
 	const vec4 bit_shift = vec4( 256.0 * 256.0 * 256.0, 256.0 * 256.0, 256.0, 1.0 );
 	const vec4 bit_mask = vec4( 0.0, 1.0 / 256.0, 1.0 / 256.0, 1.0 / 256.0 );
-	vec4 res = mod( value * bit_shift * vec4( 255 ), vec4( 256 ) ) / vec4( 255 );
+	vec4 res = fract( value * bit_shift );
```

Comment:
@bhouston For history of this, see https://github.com/mrdoob/three.js/issues/4341.